en/docs/administration/configurations/configure-logs-router.md

```diff
@@ -0,0 +1,334 @@
+# Router Log Configurations
+
+As the APK uses an Envoy proxy as the Router component, it only supports the logging mechanisms provided by the Envoy proxy. In the following sections, you may find how to enable Router access logs and debug logs.
+
+## Router Access Logging
+
+You can enable Router access logs by using the following configs. Router access logs related configurations are in the `log_config.toml` file, which is located in ConfigMap file located in the following file in the APK helm chart.
+
+```yaml
+apk/helm-charts/templates/data-plane/gateway-components/log-conf.yaml
+```
+
+Setting `enable` as `true` will let you enable the Router access logs and the file path can be set up using the `logfile`. 
+
+```yaml
+[accessLogs]
+enable = true
+logfile = "/tmp/envoy.access.log"
+format = "[%START_TIME%] '%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%' %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% '%REQ(X-FORWARDED-FOR)%' '%REQ(USER-AGENT)%' '%REQ(X-REQUEST-ID)%' '%REQ(:AUTHORITY)%' '%UPSTREAM_HOST%'\n"
+```
+
+## Router debug logs
+
+To enable Router debug logs, provide the log level as trailing arguments for the envoy command as follows.
+
+```yaml
+-l trace, 
+--log-level trace
+--component-log-level upstream:debug,connection:trace
+```
+
+You can do this by setting the `TRAILING_ARGS` environment variable in the Router. For example, Add the following line under router configuration in the gateway-runtime-deployment.yaml in the directory `apk/helm-charts/templates/data-plane/gateway-components/gateway-runtime/`.
```

Comment:
Shouldn't we also include the way in which we can add this using the values.yaml?

Reply:
We will have to support all the router related environment configurations including this variable through values.yaml, but for the initial doc, we can sticking with current implementation of providing through helm configs?